### PR TITLE
Acceptance tests for login and bidding

### PR DESCRIPTION
**Pull request by craigspaeth**
Closes https://github.com/artsy/force/issues/1355

This is the result of our pairing session yesterday which includes...

* Running acceptance tests against minified assets
* Acceptance tests for login and bidding
* Helpers like `browser.login` to extract common actions

### CODE REVIEW

**Comment by sweir27** [comment]
This looks good to me!! cc @joeyAghion since he is curious!

(Looks like it's hanging in CI).

**Comment by craigspaeth** [comment]
😞  Probably going to want to run these tests separately now that asset compilation slows it down significantly. Acceptance tests 🤷‍♂️

**Comment by sweir27** [comment]
😞

**Comment by craigspaeth** [comment]
😮  it passed! 💚

File: scripts/test.sh [hunk 1]
```diff
@@ -5,12 +5,12 @@ set -e -x
 run () {
   case $CIRCLE_NODE_INDEX in
   0)
-    yarn acceptance
-    yarn mocha test/lib/*
-    yarn mocha $(find desktop/test -name '*.coffee')
-    yarn mocha $(find mobile/test -name '*.coffee')
+    yarn assets
+    yarn acceptance test/acceptance/*.js
```
Comment (craigspaeth) [inline scripts/test.sh]: Letting acceptance tests take up one container for itself now which evens out the build time for now.

**Pull request merged**